core-common/essentials-deploy/deploy.bat

```diff
@@ -0,0 +1,56 @@
+@echo OFF
+cls
+
+:start
+set /p VERSION=Type the version:
+if [%VERSION%]==[] goto start
+
+if not exist .\work md "work"
+cd .\work
+del /Q *.*
```

Comment:
it is very dangerous to delete *.*. 

Maybe you should use 

```
rmdir /Q /S work
md work
```
instead.

Reply:
done